wire-ios/Configuration/Images.xcassets/AppIcon.appiconset/Contents.json

```diff
@@ -1,200 +1,242 @@
 {
   "images" : [
     {
-      "size" : "20x20",
-      "idiom" : "iphone",
       "filename" : "Icon-App-20x20@2x.png",
-      "scale" : "2x"
+      "idiom" : "iphone",
```

Comment:
Btw, if the icon is the same across all size variants, you could provide a single 1024x1024 graphic and Xcode will do all the scaling.

Reply:
This file wasn't added. I think these are just changes from the new Xcode for app icon

Reply:
Looks like Xcode updated the icon assets. These haven't been changed. Not in the scope